backend/Repositories/SightingRepo.cs

```diff
@@ -24,5 +27,25 @@ public IEnumerable<Sighting> GetAllSightings()
                 .Sightings
                 .Include(s => s.Species);
         }
+
+         public Sighting GetById(int id)
+        {
+            return _context
+                .Sightings
+                .Single(s => s.Id == id);
+        }
+         public Sighting DeleteSighting(int id)
+        {
+             Sighting sighting = _context
+                .Sightings
+                .Include(s => s.Species)
+                .Single(s => s.Id == id);
+            sighting.IsDeleted = true;
+            _context.Sightings.Update(sighting);
+            _context.SaveChanges();
+
+            return sighting;
```

Comment:
You shouldn't return the deleted object, it's been deleted! From the point of view of the API, this Sighting should no longer exist!